src/models/Organism.tsx

```diff
@@ -80,15 +69,15 @@ export const Organism = types
           workingReceptor = true;
           break;
       }
-      return {
+      return observable.map({
```

Comment:
As suggested by the link I posted above, making the `modelProperties` an `ObservableMap` seems like the best way to make sure changes to individual properties get picked up by MobX.

Reply:
Go for it. I'll try to actually follow a MobX tutorial at some point.